client/block.c

```diff
@@ -353,7 +378,22 @@ static int valid_signature(const struct xdag_block *b, int signo_r, int keysLeng
 	if (signo_s >= 0) {
 		for (i = 0; i < keysLength; ++i) {
 			hash_for_signature(buf, keys + i, hash);
+#if OPENSSL == 1
+			if (!xdag_verify_signature(keys[i].key, hash, b->field[signo_r].data, b->field[signo_s].data)) {
+#elif OPENSSL == 0
+			if (!xdag_verify_signature_noopenssl(keys[i].pub, hash, b->field[signo_r].data, b->field[signo_s].data)) {
+#elif OPENSSL == 2
```

Comment:
I suggest you to leave two options - standart verifying and optimizaed verifying

Reply:
it's just for testing purpose, if someone can run the whole dag with that option, can tell if there are issues.

Reply:
Ok, we can leave it for testing purposes, but it should be removed before release.